config.py

```diff
@@ -0,0 +1,74 @@
+import configparser
+from os import path
+
+
+class Config:
+    def __init__(self, section):
+        conf_file = path.join(path.dirname(__file__), 'config.ini')

```

Comment:
Please, drop this line)
and re-white 9 line - 
self._config_file = path.join(path.dirname(__file__), 'config.ini')